packages/shared/src/lib/localLanguage.ts

```diff
@@ -49,25 +23,18 @@ const localLanguages: LocalLanguages = {
     },
 };
 
-export const countryCodeToVerfiedLocalLanguage = (
+export const countryCodeToLocalLanguageBannerHeader = (
     testName: string,
     variantName: string,
     countryCode?: string,
-    dfltLocalLanguage?: LocalLanguage,
+    defaultBannerHeader?: LocalLanguage,
 ): LocalLanguage | undefined => {
```

Comment:
With the refactor the return type should not need the inclusion of `... | undefined`

Reply:
Yes, removed, always returning the defaultBannerHeader